db/trial.sql

```diff
@@ -0,0 +1,7 @@
+\c nc_news_test
```

Comment:
worth adding files like this to your .gitignore so they aren't getting pushed up unnecessarily